### PR TITLE
added longer padding for section

### PR DESCRIPTION
**Pull request by claireboyd**
added longer padding on the section and i think this solved the issue - not exactly sure why its cutting off, but we can play around with it more when we have the doc page working with the NLP output.

### CODE REVIEW

File: civiclens/templates/document.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 {% block content %}
 
 <!-- whole page -->
-<section class="slice py-3" style="background-color:#FDFFF7" id="core-block">
+<section class="slice py-3 pb-6" style="background-color:#FDFFF7" id="core-block">
```
Comment (jchristensonCHI) [inline civiclens/templates/document.html:11]: <section class="slice py-3 pb-10" style="background-color:#FDFFF7" id="core-block">

Comment (jchristensonCHI) [inline civiclens/templates/document.html:11]: `<section class="slice py-3 pb-10" style="background-color:#FDFFF7" id="core-block">`

I checked this on the code I made a PR on and 6 was an improvement, but still had issues, on 5 that I quickly checked changing this to 10 seemed to work

**Review by jchristensonCHI — CHANGES_REQUESTED** [review]

File: civiclens/templates/document.html (hunk 1 above)
Comment (claireboyd) [reply civiclens/templates/document.html:11]: yeah it's super weird- this is definitely not solving the root cause of the issue, but could be enough to get us through tomorrow.

Do you think i should spend a bit more time looking into it or stick with pb-10 for now?

Comment (jchristensonCHI) [reply civiclens/templates/document.html:11]: We can stick to pb 10 for now and reflag it if it's a noticeable issue when the backend bros start testing everything out

**Review by jchristensonCHI — APPROVED** [review]

**Pull request merged**